sample_events/shared_link_created.json

```diff
@@ -0,0 +1,133 @@
+{
```

Comment:
is everything in the sample events safe for public consumption?

Reply:
Excellent question, I'd better censor these

Reply:
(got rid of the Box signature fields, just in case)